Sources/PointFree/Admin/Index.swift

```diff
@@ -87,19 +87,17 @@ private func sendNewEpisodeEmails<I>(_ conn: Conn<I, Episode>) -> IO<Conn<I, Pre
 private func sendEmail(forNewEpisode episode: Episode, toUsers users: [Database.User]) -> EitherIO<Prelude.Unit, Prelude.Unit> {
 
   return lift <| IO {
-    // TODO: look into mailgun rate limits. we could batch subscribers and non subscribers at least
     let newEpisodeEmails = users.enumerated().map { idx, user in
       sendEmail(
         to: [user.email],
         subject: "New Point-Free Episode: \(episode.title)",
-        content: inj2(newEpisodeEmail.view((episode, true)))
+        content: inj2(newEpisodeEmail.view((episode, user, true)))
```

Comment:
user gets passed along to this email cause it's used to construct the unsubscribe link